app/src/main/graphql/com/cornellappdev/android/eatery/schema.json

```diff
@@ -1273,6 +1293,26 @@
             "isDeprecated": false,
             "deprecationReason": null
           },
+          {
+            "name": "exceptions",
+            "description": null,
+            "args": [],
+            "type": {
+              "kind": "NON_NULL",
+              "name": null,
+              "ofType": {
+                "kind": "LIST",
+                "name": null,
+                "ofType": {
+                  "kind": "SCALAR",
+                  "name": "String",
+                  "ofType": null
+                }
+              }
+            },
+            "isDeprecated": false,
+            "deprecationReason": null
+          },
```

Comment:
I guess we'll stick with this for now. Feel like we should be getting an updated Schema.json from the backend, but if it works It works.